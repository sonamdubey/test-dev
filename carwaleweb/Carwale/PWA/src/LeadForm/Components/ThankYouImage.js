import React from "react";

function ThankYouImage() {
  return (
    <div className="thankyou-screen">
      <div className="thankyou-text">
        A car consultant would get in touch with you shortly to assist on your
        purchase.
      </div>
    </div>
  );
}

export default ThankYouImage;
