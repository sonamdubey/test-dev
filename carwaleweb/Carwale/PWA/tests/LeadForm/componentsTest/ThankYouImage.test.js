import React from "react";
import ThankYouImage from "../../../src/LeadForm/Components/ThankYouImage";

test("Component ThankYouImage Snapshot RenderedTree", () => {
  let wrapper = shallow(<ThankYouImage />);
  expect(wrapper).toMatchSnapshot();
});
