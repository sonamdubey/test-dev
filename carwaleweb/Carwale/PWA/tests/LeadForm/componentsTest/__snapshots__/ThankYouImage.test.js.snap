// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component ThankYouImage Snapshot RenderedTree 1`] = `
<div
  className="thankyou-screen"
>
  <div
    className="thankyou-text"
  >
    A car consultant would get in touch with you shortly to assist on your purchase.
  </div>
</div>
`;
